### PR TITLE
add fasttext module

### PR DESCRIPTION
**Pull request by erip**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This is a fix to be merged in lieu of seemingly dead PR #10.

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -2,3 +2,5 @@
 make -j"${CPU_COUNT}" CXX="${CXX}"
 mkdir -p "${PREFIX}/bin"
 mv fasttext "${PREFIX}/bin"
+
+${PYTHON} -m pip install --no-deps --ignore-installed -v .
```
Comment (kylebgorman) [inline recipe/build.sh]: quote variable expansion---match style of the rest of the script please

**Review by kylebgorman — APPROVED** [review]
LGTM

**Comment by erip** [comment]
@kylebgorman it looks like there's something funny happening with the OS X build so I'll try to resolve that. I somehow don't think scipy is actually required...

**Comment by erip** [comment]
OK, @kylebgorman I think this is good now.

**Pull request merged**